mautic/mautic
# [UX] Configuration tips and extra context

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR adds context for settings fields and changes the page appearance to reduce information density. 
Specifically re. context, while tooltips are more functionality-descriptive, these help texts aim to be more like a smart tip or just provide the necessary information so the user have idea of what this setting is about without searching docs.

It also adds a panel to help people set their date format.

![image](https://github.com/user-attachments/assets/bfd88720-7178-4597-a80f-7faf62b35949)





| Before                                 | After
| -------------------------------------- | ---
|       ![Screenshot From 2024-12-21 13-47-11](https://github.com/user-attachments/assets/63a39639-d2bc-4d5c-9088-2f7df24450e0)                                 | ![Screenshot From 2024-12-21 13-46-12](https://github.com/user-attachments/assets/a6cbe5d7-6436-4df1-8fb2-f3b99f63bc2c)



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Settings > Have a look at all tabs
